css/index.css

```diff
@@ -143,4 +214,47 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+.top-content-about {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+}
+
+.top-content-about .text-container-about {
+    padding: 15px 1%;
+    padding-bottom: 20px;  
+}
+
+.middle-about-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+}
+
+.middle-about-content .about-box {
+    width: 48%;
+    border-bottom: 1px dotted black;
+    padding: 15px 0;
+    margin: 10px 0;
+}
+
+.middle-about-content .about-box h2 {
+    margin: 15px 0;
+}
+
+.middle-about-content .about-box button {
+    margin: 15px 0;
+    padding: 10px 20px;
+    border-radius: 10px;
+    border: 1px solid black;
+}
+
+.about-buttom-content {
+    margin: 25px 0;
+    margin-left: 15px;
```

Comment:
usually you don't want to be setting margin in two different properties on the same selector. This can lead to buggy code later on if you forget there are two of them and are surprised that your changes aren't rendering properly.